lnbits/extensions/subdomains/templates/subdomains/display.html

```diff
@@ -0,0 +1,173 @@
+{% extends "public.html" %} {% block page %}
+<div class="row q-col-gutter-md justify-center">
+  <div class="col-12 col-md-7 col-lg-6 q-gutter-y-md">
+    <q-card class="q-pa-lg">
+      <q-card-section class="q-pa-none">
+        <h3 class="q-my-none">{{ domain_domain }}</h3>
+        <br />
+        <h5 class="q-my-none">{{ domain_desc }}</h5>
+        <br />
+        <q-form @submit="Invoice()" class="q-gutter-md">
+          <q-input filled dense v-model.trim="formDialog.data.email" type="email"
+            label="Your email (optional, if you want a reply)"></q-input>
+          <q-input filled dense v-model.trim="formDialog.data.subdomain" type="text" label="subdomain you want">
+          </q-input>
+          <q-input filled dense v-model.trim="formDialog.data.ip" type="text" label="ip of your server">
+          </q-input>
+          <q-input filled dense v-model.trim="formDialog.data.duration" type="number" label="{{ domain_cost }} sats per day">
```

Comment:
This field is confusing, what is the user supposed to write here? The number of days?

Reply:
yeah number of days.. will make it clearer